client/landing/stepper/declarative-flow/internals/steps-repository/site-picker/index.tsx

```diff
@@ -24,14 +24,17 @@ const SitePickerStep: Step = function SitePickerStep( { navigation } ) {
 		DEFAULT_SITE_LAUNCH_STATUS_GROUP_VALUE;
 
 	const onQueryParamChange = ( params: Partial< SitesDashboardQueryParams > ) => {
+		recordTracksEvent( 'calypso_import_site_picker_query_param_change', params );
 		navigation.submit?.( { action: 'update-query', queryParams: params } );
 	};
 
 	const createNewSite = () => {
+		recordTracksEvent( 'calypso_import_site_picker_create_new_site' );
 		navigation.submit?.( { action: 'create-site' } );
 	};
 
 	const selectSite = ( site: SiteExcerptData ) => {
+		recordTracksEvent( 'calypso_import_site_picker_select_site', site );
```

Comment:
Saw the following error when click `select this site`, I think we probably don't need to record the whole `site` object here. 
<img width="1195" alt="Screen Shot 2023-05-23 at 1 24 39 PM" src="https://github.com/Automattic/wp-calypso/assets/4074459/18fc5bb8-3046-4228-98c7-b3035e019b08">

Reply:
Fixed!
Thanks for reporting it.